RELEASING.md

```diff
@@ -12,32 +12,31 @@ See the [prerequisites](#prerequisites) if this is your first release.
 
 * [ ] [Check CI][ci] passes against the latest Scala versions.
 * [ ] [Draft a new release][releases/new] for the release, allowing GitHub to generate draft release notes automatically.
-* [ ] Bump the version in the `README.md` and `git commit -am 'Release 1.x.y`.
-* [ ] Run `git tag -a -s 1.x.y -m 'Version 1.x.y`. The name of the tag should NOT have a 'v' prefix.
+* [ ] Run `git tag -a -s 1.x.y -m 'Version 1.x.y'`. The name of the tag should NOT have a `v` prefix.
 * [ ] In sbt run `reload` and `show version` to verify the version.
 * [ ] In sbt run `clean`, particularly if you've recently bumped `scalaVersion`.
 * [ ] In sbt run `publishSigned`. You should start seeing "published mima-.. to https://oss.sonatype.org/service/local/staging/deploy/maven2/..".
 * [ ] In sbt run `++2.13` `coreJVM/publishSigned` `coreNative/publishSigned`
 * [ ] In sbt run `++3.2` `coreJVM/publishSigned` `coreNative/publishSigned`
 * [ ] [Find and close][sonatype/staging-repos] your staging repository.  (See Sonatype's [Releasing the Deployment][sonatype/guide] guide.)
 * [ ] Switch to a branch (e.g. `git checkout -b release-1.x.y`)
-* [ ] In `plugins.sbt` update `sbt-mima-plugins`.
+* [ ] In `project/plugins.sbt` update `sbt-mima-plugin`.
 * [ ] In `project/MimaSettings.scala` update `mimaPreviousVersion` & clear out `mimaBinaryIssueFilters`.
 * [ ] In sbt run `testStaging` **WITHOUT** `reload`ing first (`testStaging` adds the staging resolvers & runs `reload`).
-* [ ] Run `git commit -am 'Update sbt-mima-plugin to 1.x.y` and PR it (`hub pull-request`). The PR won't pass CI until the release is available on Maven Central. You may poll [repo1 directly][repo1/list] (note the trailing slash in the URL).
+* [ ] Run `git commit -am 'Update sbt-mima-plugin to 1.x.y'` and PR it (`gh pr create` or `hub pull-request`). The PR won't pass CI until the release is available on Maven Central. You may poll [repo1 directly][repo1/list] (note the trailing slash in the URL).
```

Comment:
Btw, they were all intentionally unbalanced quotes so I didn't accidentally commit those stub messages every time.  Then I never actually forgot, because I kept remember this trick.  Meaning it served its purpose, for me, the unexpected way. 😄

Reply:
Ha, I see! Well, I'm volunteering to do the releases, so I get to have it my way :-)